CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+Community patches, bug reports, and contributions are always welcome and are crucial to ensure `google-music.js` stays stable. Here are steps/guidelines to follow when contributing.
+
+## Opening an issue
+Before starting work on an issue, please make sure that the issue isn't being worked on by anyone else. This can be verified by searching in open issues and pull requests:
+
+https://github.com/twolfson/google-music.js/issues
+
+- If an issue already exists, please use that issue instead of creating a new one
+- Otherwise, please create a new issue
+    - Please restrict issues to a single feature/issue
+    - If the issue is a bug report, then please provide information about your setup as well as steps to reproduce
+- If the issue is time sensitive, then feel free to start work immediately
+    - Please communicate that via the issue (e.g. via a comment on the thread)
+- Otherwise, please wait for a core contributor to approve your work
+    - This is prevent unnecessary rejection when a pull request is opened
+
+## Making changes
+To make sure that there is no duplicate work across contributors, please always [Open an issue][] before beginning work.
```

Comment:
Weird. When I clicked it it didn't do anything. May be because I'd already clicked it once before :-/